cloud-arm/build.gradle

```diff
@@ -0,0 +1,38 @@
+apply plugin: 'java'
+
+sourceCompatibility = 1.7
+version = '1.0'
+
+repositories {
+    maven { url 'http://repo.spring.io/libs-release' }
+    mavenCentral()
+}
+
+jar {
+    baseName = 'cloud-arm'
+}
+
+dependencies {
+
+    compile project(':cloud-api')
+    compile project(':cloud-common')
+    testCompile project(':cloud-reactor')
+
+    compile group: 'org.slf4j',                     name: 'slf4j-api',                      version: slf4jApiVersion
+    compile group: 'org.springframework',           name: 'spring-context-support',         version: '4.0.3.RELEASE'
+    compile group: 'org.springframework.boot',      name: 'spring-boot-starter',            version: springBootVersion
+    compile group: 'io.projectreactor',             name: 'reactor-core',                   version: eventBusVersion
+    compile group: 'io.projectreactor',             name: 'reactor-bus',                    version: eventBusVersion
```

Comment:
- reactor dependency not needed
- freemarker is duplicated, would and also would be better to externalise the version number
- guava is used in another modul, it is better to introduce a common variable for it
- compile ("com.sequenceiq:azure-rest-client:${azureRestClientVersion}") { exclude group: 'log4j' } has a different syntac than others
- mocikto shall be included as test dependency
- junit shall be included as test dependency